hieradata/clients/aws.ci.jenkins.io.yaml

```
@@ -90,6 +90,134 @@ profile::jenkinscontroller::jcasc:
     targetHost: "172.17.0.1" # docker0 interface
     collectBuildLogs: true
   cloud_agents:
+    kubernetes:
+      ci.jenkins.io-agents-2:
+        enabled: true
+        acpServerId: azure-aks-internal # will probably be aws-eks-internal
```

```suggestion
        acpServerId: aws-eks-internal
```

Since https://github.com/jenkins-infra/jenkins-infra/pull/3827